src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,11 +5,37 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private double excavatorReach;
+    private double bucketCapacity;
+
     public Excavator() {
     }
 
+    public Excavator(double excavatorReach, double bucketCapacity) {
+        this.excavatorReach = excavatorReach;
+        this.bucketCapacity = bucketCapacity;
+    }
+
     @Override
     public void doWork() {
         System.out.println("Excavator started to work");
     }
+
+    public double getExcavatorReachInMeters() {
+        return excavatorReach;
+    }
+
+    public Excavator setExcavatorReachInMeters(int excavatorReachInMeters) {
+        this.excavatorReach = excavatorReachInMeters;
+        return this;
+    }
+
+    public double getBucketCapacity() {
+        return bucketCapacity;
+    }
+
+    public Excavator setBucketCapacity(int bucketCapacity) {
```

Comment:
The parameter 'bucketCapacity' should match the field type 'double'. The method currently accepts an 'int', which could lead to unnecessary casting or precision loss.